src/node-lib/curriculum-api-2023/queries/unitListing/unitListing.gql

```diff
@@ -1,9 +1,6 @@
-query unitListing($programmeSlug: String!, $isLegacy: Boolean!) {
+query unitListing($programmeSlug: String!) {
   programme: published_mv_unit_listing_page_5_0_0(
```

Comment:
do we not need to update to the latest MV to get core?

Reply:
✅